src/utils/formating.ts

```diff
@@ -0,0 +1,3 @@
+export const contenfulImageFormated = (imageObject: any): Object => {
+  return { alt: imageObject?.fields.title, url: imageObject?.fields.file?.url };
```

Comment:
This is absolutely fine... but let's think about a way to abstract with a more generalist format.

Do we need to transform images from Contentful or Posts? We can answer this question together in today's session.